### ISSUE TITLE
error when attempting to disable account password expiration

### ISSUE THREAD

**Issue by zaro0508** [org-formation/org-formation-cli#145]
I want to disable password expiration however I get an error when I removing MaxPasswordAge from OC::ORG::PasswordPolicy.  The doc says that `MaxPasswordAge` is optional, https://github.com/org-formation/org-formation-cli/blob/master/docs/organization-resources.md#passwordpolicy

steps to repro:
1.  run org-formation update with password policy
```
  PasswordPolicy:
    Type: OC::ORG::PasswordPolicy
    Properties:
      MaxPasswordAge: 30
      MinimumPasswordLength: 12
      RequireLowercaseCharacters: true
      RequireNumbers: true
      RequireSymbols: true
      RequireUppercaseCharacters: true
      PasswordReusePrevention: 5
      AllowUsersToChangePassword: true
```
2.  remove MaxPasswordAge
3.  run org-formation update again
```
➜  org-formation update org-formation/organization.yml --profile my-profile --verbose
ERROR: unable to load file org-formation/organization.yml.
reason: expected PasswordReusePrevention to be a number, found undefined (undefined).
```

Note - i can remove MinimumPasswordLength and update without error.

**Comment by eduardomourar** [comment]
The specific orgformation version might help us with debugging.

We have been working a lot on actually moving those resource types to another repository whenever they are not particular to orgformation.

**Comment by zaro0508** [comment]
```
➜  org-formation --version
0.9.13
```

the docs in that location says MaxPasswordAge is not required as well

**Comment by eduardomourar** [comment]
For you to use the cloudformation native resource type, you would have to first register it then use ``. The one mentioned by you is making the API calls directly to AWS.

**Comment by OlafConijn** [comment]
hi @zaro0508

thanks for reporting this. i looked into it and it seems the underlying api will remove the reuse prevention if the value `0` is passed as an argument. the documentation seems to state something different, but in my test this was the case.

this means that:
1) quick workaroud would be passing value `0` and this will remove the reuse-prevention.
2) fix was just committed and will be available in v0.9.14
3) indeed, as eduardo mentioned, it might be worth the while to look at the types from the org-formation-resource-providers project. there is a lot more interesting types there that help with sensible account details (or other IaC problems).


hoped this helped!
Olaf
